pkgs/fc/agent/default.nix

```diff
@@ -69,6 +69,6 @@ py.buildPythonPackage rec {
     xfsprogs
   ];
   dontStrip = true;
-  passthru.pythonDevEnv = py.withPackages (_: checkInputs ++ propagatedBuildInputs);
+  passthru.pythonDevEnv = python310.withPackages (_: checkInputs ++ propagatedBuildInputs);
```

Comment:
small nitpick: right now we use the name indirection `py` to pin packages to a certain version with necessary. When changing the pinning, only that name binding needs to be specified.
Now you introduce the explicit specification of a python release a 2nd time by directly specifying `python310` here.
It'd be cleaner to stack name indirections like `py` and `pyPkgs` or `py.packages`. Or at least also use another alias name here for `python310`, that is defined as a let binding directly next to `py` to keep version specifications next to each other.